justfile

```diff
@@ -28,13 +30,13 @@ md-lint:
 
 # Fetch dependencies
 fetch:
-    @just-cargo fetch
+    @just-cargo target={{ target }} fetch
 
 check *args:
     @just-cargo check --workspace --all-targets {{ _features }} {{ args }}
 
 clippy-all *args:
-    @just-cargo clippy --workspace --all-targets --all-features {{ args }}
+    @just-cargo target={{ target }} clippy --workspace --all-targets --all-features {{ args }}
```

Comment:
we need to make this consistent -- if we need it in clippy-all, we should have it on clippy (and check, etc)